facets-wallpaper.js

```diff
@@ -0,0 +1,31 @@
+var cheerio = require('cheerio'),
+	request = require('request'),
+	fs = require('fs');
+
+var savePath = process.argv[2];
+
+if(typeof savePath === 'undefined') {
+	console.log("Error: Save path not defined.");
+	return;
+}
+
+var year = "2013";
+var baseUri = 'http://www.facets.la/';
+
+(function getImg(id) {
+	var uri = baseUri + year + '/' + id + '/wallpaper/';
```

Comment:
Perhaps instead creating a new file for this, we could have an option passed as a command line argument to denote if wallpaper or standard image downloads are desired. That way we can reuse this code.

For example: 

`node facets-downloader.js SAVE_PATH_HERE wallpaper`